client/my-sites/plugins/plugins-browser-list/index.jsx

```diff
@@ -125,18 +108,14 @@ const PluginsBrowserList = ( {
 					) }
 				</div>
 			</div>
-			{ listName === 'paid' &&
-				isEnabled( 'marketplace-spotlight' ) &&
-				spotlightPluginFetched &&
-				spotlightPlugin && (
-					<Spotlight
-						taglineText={ __( 'Drive more traffic with Yoast SEO Premium' ) }
-						titleText={ __( 'Under the Spotlight' ) }
-						ctaText={ __( 'View Details' ) }
-						illustrationSrc={ spotlightPlugin?.icon ?? '' }
-						onClick={ () => spotlightOnClick() }
-					/>
-				) }
+			{ listName === 'paid' && isEnabled( 'marketplace-spotlight' ) && (
+				<AsyncLoad
+					require="calypso/blocks/jitm"
+					template="spotlight"
+					placeholder={ null }
+					messagePath="calypso:plugins:spotlight"
```

Comment:
Why do we async load a component that async loads another? Is there some rendering thing this hides or something like that?

Reply:
Good question, actually I followed that pattern that is used everywhere I find a jitm block declaration. would like to understand a little more why this is done that way been researching previous commit this seems to be the first time it was used https://github.com/Automattic/wp-calypso/pull/37332

Reply:
maybe @taggon can chime in on this one ?

Reply:
Thanks for digging, definitely not a blocker for merging.